notebooks/DrizzlePac/Initialization/Initialization.ipynb

```diff
@@ -0,0 +1,319 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obs_table = Observations.query_criteria(obs_id='ib2j02n5q', obstype='all')\n",
+    "download_tab = Observations.download_products(obs_table['obsid'], mrp_only=False, \n",
+    "                                              productSubGroupDescription=['FLC'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's move all of the files we just downloaded to the current working directory and remove the \"mastDownload/\" directory:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for file in download_tab['Local Path']:\n",
+    "    os.rename(file, os.path.basename(file))\n",
+    "    \n",
+    "shutil.rmtree('mastDownload')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reference Files\n",
+    "\n",
+    "HST data require reference files that specify calibration information. There are many types of reference files for each detector, so care should be taken to ensure that all necessary reference files for a particular dataset have been retrieved. The path to the reference files for each instrument (e.g., WFC3, ACS, WFPC2) is indicated with the appropriate environment variable. For the instruments supported by DrizzlePac, these are:\n",
+    "\n",
+    "- ACS = jref\n",
+    "- WFC3 = iref\n",
+    "- WFPC2 = uref\n",
+    "\n",
+    "The Calibration Reference Data System (CRDS) both stores the reference files and determines the mapping of reference files to observations. The `crds` tool included in AstroConda can find and download the best reference files for a particular observation. The [documentation](https://hst-crds.stsci.edu/static/users_guide/index.html) for `crds` describes many of the more advanced options, but we will demonstrate here how to obtain update reference file information stored in the FITS header of an observation and also download those files to a local directory.\n",
+    "\n",
+    "First we need to set some environment variables:\n",
+    "- CRDS_SERVER_URL: Location of the CRDS server.\n",
+    "- CRDS_PATH: Path to where reference files will be downloaded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'\n",
+    "os.environ['CRDS_PATH'] = './reference_files'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the `crds.bestrefs` tool is also accessible inside of Python, it was designed with a command line interface in mind, therefore it is easiest to use it this way:"
```

Comment:
```suggestion
     "While the `crds.bestrefs` tool is also accessible inside of Python, it was designed with a command line interface in mind, therefore it is easiest to use it this way.  The ``!`` in front of the cell below does this by executing the command as though it was run in a terminal instead of in python:"
```
because the `!` business is not a commonly-known Jupyter feature.